sycl/include/sycl/detail/pi.h

```diff
@@ -60,9 +60,14 @@
 // PI_EXT_ONEAPI_QUEUE_PRIORITY_HIGH queue properties.
 // 11.18 Add new parameter name PI_EXT_ONEAPI_QUEUE_INFO_EMPTY to
 // _pi_queue_info.
+// 12.19 Add new PI_DEVICE_EXT_INTEL_PARTITION_BY_CSLICE piDevicePartition
+// scheme. Sub-sub-devices (representing compute slice) creation via
+// partitioning by affinity domain is disabled by default and can be temporarily
+// restored via SYCL_PI_LEVEL_ZERO_EXPOSE_CSLICE_IN_AFFINITY_PARTITIONING
+// environment variable.
 
-#define _PI_H_VERSION_MAJOR 11
-#define _PI_H_VERSION_MINOR 18
+#define _PI_H_VERSION_MAJOR 12
```

Comment:
> In the cases where backward compatibility is not maintained there should be a (+1) change to the major version

Why did you change the major version?

Reply:
On PVC device it was possible to get sub-sub-device by partitioning by affinity domain by default before this PR and now it requires either using different partitioning scheme or non-default environment variable.